app/precompiled.py

```diff
@@ -240,14 +240,14 @@ def overlay_template_areas(src_pdf, page_number=None, overlay=True):
 def get_invalid_pages_with_errors(src_pdf):
     passed_validation = False
     errors = {
-        "pages_out_of_printable_area": [],
-        "non_a4_portrait_pages": [],
+        "content_outside_of_printable_area": [],
+        "document_not_a4_size_portrait_orientation": [],
```

Comment:
in fact, after looking through i prefer the old values as keys - makes it clear it's talking about pages. (appreciate you changed this for admin's sake, but i'd rather admin just had a mapping dict.